quality_control_stock_oca/tests/test_quality_control_stock.py

```diff
@@ -97,7 +97,7 @@ def setUp(self):
     def _create_user(self, login, groups, company):
```

Comment:
@victoralmau, how is the existing helper method in core for doing this?

Reply:
https://github.com/odoo/odoo/blob/15.0/odoo/tests/common.py#L110-L154

Reply:
@dalonsod please use this method instead.